src/VivyFooter/VivyFooter.jsx

```diff
@@ -4,45 +4,47 @@ import { Col, Row, Footer } from "../index";
 
 export default function VivyFooter() {
   return (
-    <Footer>
-      <Row>
-        <Col lg={12}>
-          <Styles>
-            <Row position="center" verticalAlign="middle">
-              <Col lg={0}>
-                <a
-                  className="anchor"
-                  href="https://www.vivy.com/impressum"
-                  target="_"
-                  rel="noopener noreferrer"
-                >
-                  Impressum
-                </a>
-              </Col>
-              <Col lg={0}>
-                <a
-                  className="anchor"
-                  href="https://www.vivy.com/nutzungsbedingungen"
-                  target="_"
-                  rel="noopener noreferrer"
-                >
-                  AGB
-                </a>
-              </Col>
-              <Col lg={0}>
-                <a
-                  className="anchor last"
-                  href="https://www.vivy.com/datenschutz"
-                  target="_"
-                  rel="noopener noreferrer"
-                >
-                  Datenschutz
-                </a>
-              </Col>
-            </Row>
-          </Styles>
-        </Col>
-      </Row>
-    </Footer>
+    <div style={{ marginTop: 100 }}>
```

Comment:
I'm wondering if this is better to be specified in the parent component (the page) rather than in the footer. What do you think?

Reply:
thats my bad. I thought actually this is the story that I was adding the margin not the component itself. Will change that in one sec. Thanks